be/src/olap/rowset/segment_v2/segment_iterator.cpp

```diff
@@ -379,7 +379,17 @@ Status SegmentIterator::_get_row_ranges_from_conditions(RowRanges* condition_row
         DCHECK(_opts.col_id_to_predicates.count(cid) > 0);
         RETURN_IF_ERROR(_column_iterators[_schema.unique_id(cid)]->get_row_ranges_by_zone_map(
                 _opts.col_id_to_predicates[cid].get(),
-                _opts.col_id_to_del_predicates.count(cid) > 0
+                // Currently, one transaction can only support one delete condition for page filtering.
```

Comment:
It seems that when delete condition only contains OR predicates, page filter can work correctly here.
So this pr is a quick fix, it may cause some case performance degradation.
I think it's better to add a todo here.

Reply:
ok, i add a todo